service/schema/create.ddl

```diff
@@ -21,77 +20,29 @@ CREATE TABLE IF NOT EXISTS core."entity"(
+/* Section 1 - set up Journey */
 
-CREATE TABLE IF NOT EXISTS core."department"(
-  "departmentId" SMALLSERIAL PRIMARY KEY,
+CREATE TABLE IF NOT EXISTS core."journeyStatus"(
+  "journeyStatusId" SMALLSERIAL PRIMARY KEY,
   "name" VARCHAR(32) NOT NULL,
-  "descr" VARCHAR(64) NULL
+  "descr" VARCHAR(128) NULL
 );
-COMMENT ON TABLE core."department" IS 'Groups campaigns by department';
 
-CREATE TABLE IF NOT EXISTS core."campaign"(
-  "campaignId" SMALLSERIAL PRIMARY KEY,
-  "departmentId" INT NULL REFERENCES core."department"("departmentId"),
-  "name" VARCHAR(32) NOT NULL,
-  "descr" VARCHAR(64) NULL,
-  "created" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  "deleted" TIMESTAMP NULL
-);
-COMMENT ON TABLE core."campaign" IS 'Groups campaign revisions';
 
-CREATE TABLE IF NOT EXISTS core."campaignRev"(
-  "campaignRevId" SERIAL PRIMARY KEY,
-  "parentRevId" INT NULL REFERENCES core."campaignRev"("campaignRevId"),
-  "campaignId" SMALLINT NOT NULL REFERENCES core."campaign"("campaignId"),
-  "descr" VARCHAR(64) NULL,
+CREATE TABLE IF NOT EXISTS core."journey"(
+  "journeyId" SERIAL PRIMARY KEY,
   "created" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  "isActive" BOOL NOT NULL DEFAULT TRUE,
-  "deleted" TIMESTAMP NULL
-);
-COMMENT ON TABLE core."campaignRev" IS 'Groups together a set of tracks the entity can follow,
-or "entity/user journeys"';
-
-CREATE TABLE IF NOT EXISTS core."trackStatus"(
-  "trackStatusId" SMALLSERIAL PRIMARY KEY,
-  "name" VARCHAR(32) NOT NULL,
-  "descr" VARCHAR(128) NULL
-);
-
-CREATE TABLE IF NOT EXISTS core."track"(
-  "trackId" SERIAL PRIMARY KEY,
-  "campaignRevId" INT NOT NULL REFERENCES core."campaignRev"("campaignRevId"),
+  "journeyStatusId" SMALLINT NOT NULL REFERENCES core."journeyStatus"("journeyStatusId") DEFAULT 2,
+  "parentJourneyId" INT NULL REFERENCES core."journey"("journeyId"),
   "entityTypeName" VARCHAR(16) NOT NULL REFERENCES core."entityType"("name"),
-  "trackStatusId" SMALLINT NOT NULL REFERENCES core."trackStatus"("trackStatusId") DEFAULT 2,
-  "name" VARCHAR(32) NOT NULL,
+  "name" VARCHAR(64) NOT NULL,
   "descr" VARCHAR(128) NULL,
-  "deleted" TIMESTAMP NULL
-);
-COMMENT ON TABLE core."track" IS 'Groups together differing versions of entity
-tracks (or journeys)';
-COMMENT ON COLUMN core."track"."entityTypeName" IS 'A track must be restricted to one entityType';
-
-COMMENT ON TABLE core."trackStatus" IS 'Status can be pending,primed,live,paused,stopped';
-
-CREATE TABLE IF NOT EXISTS core."trackRev"(
-  "trackRevId" SERIAL PRIMARY KEY,
-  "parentTrackRevId" INT NULL REFERENCES core."trackRev"("trackRevId"),
-  "trackId" INT NOT NULL REFERENCES core."track"("trackId"),
-  "descr" VARCHAR(64) NULL,
-  "docRef" VARCHAR(128) NULL,
-  "created" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  "isActive" BOOL NOT NULL DEFAULT TRUE,
   "deleted" TIMESTAMP NULL,
   "voltQuery" TEXT NULL,
-  "GUIData" JSON
+  "GUIData" JSON,
+  "journeyConfig" JSON
```

Comment:
Can we make this column type JSONB? (It's newer, hence not used in the column above, but it is better!)

Reply:
Will do!